tests/__main__.py

```diff
@@ -44,6 +48,9 @@ def convert_argv(argv):
             lint = True
             runtests = False
             break
+        elif arg.startswith('--junit-xml='):
```

Comment:
TODO comment about the case where "=" is not used?

...and what happened to using getopt? :)

Reply:
LOL - you are reviewing my code as I'm working through it. I'm switching it to `argparse` as we speak :)